content/en/blog/falcosidekick-2-27-0-ui-2-1-0/index.md

```diff
@@ -0,0 +1,170 @@
+
+```yaml
+templatedfields: # templated fields are added to falco events and metrics, it uses Go template + output_fields values
+  Dkey: '{{ or (index . "k8s.ns.labels.foo") "bar" }}'
+```
+
+##### Hostname field
+
+Since Falco 0.33, a new field is present in Falco events: `hostname`. Falcosidekick and all its current outputs are up to date and ready for it. Once again, thanks to [Lowaiz](https://github.com/Lowaiz).
+
+##### Loki format and Grafana Cloud
+
+The Loki format has been upgraded and credentials can be set, it allows you to use [Grafana Cloud](https://grafana.com/products/cloud/) as target.
+
+##### K8S Policy Reports are binded to the namespaces
+
+Policy Reports in K8S are still prototypes but Falcosidekick is already able to create them. Some improvments have been made to determine the target resource and the report are now crated in the same namespace than the source pod.
+
+##### More headers in SMTP payload
+
+To avoid be flagged as spams by some antispams, some headers like `From`, `To` and `Date` have been added to the emails created by Falcosidekick.
+
+##### CEF format Syslog
+
+For the Syslog output, you can choose between `json` and `CEF` as format. It makes easier the integration with some services like [Microsoft Sentinel](https://azure.microsoft.com/en-us/products/microsoft-sentinel/#overview) or [Splunk](https://www.splunk.com/).
+
+### Fixes
+
+Even if we do our best to avoid them, the community faced bugs we fixed in this release. 
+
+The only important was a race condition when headers were added to the POST requests. Adopters with high rates of requests may faced authentication failures or missing headers. [bc-sb](https://github.com/bc-sb) solved this by a temproray solution we'll improve in the feature (Falcosidekick v3? Who knows...).
+
+## Falcosidekick UI v2.1.0
+
+The new features for Falcosidekick UI, even if less numerous, are as big. The full changelog is [here](https://github.com/falcosecurity/falcosidekick-ui/blob/master/CHANGELOG.md).
+
+### Env vars for settings
+
+All settings to configure Falcosidekick UI can be passed as command line arguments and env vars. Run `falcosidekick-ui --help` for more details.
+
+### New logs
+
+The logs were too verbose for production contexts, it's now configurable through a log level:
+
+```bash
+-l string   
+	  Log level: "debug", "info", "warning", "error" (default "info",  environment "FALCOSIDEKICK_UI_LOGLEVEL")
+```
+
+### Auto refresh
+
+For the long term adopters, you may remember the dashboard in v1 was auto-refreshed. The feature is back, and for all widgets, whatever the page.
```

Comment:
```suggestion
The long-term adopters may remember the dashboard in Falcosidekick UI v1 was auto-refreshed. The feature is back, for all widgets, independently of the page.
```

Reply:
Long-term adopters may remember the dashboard in Falcosidekick UI v1 was auto-refreshed. This feature is back, for all widgets, independently of the page.